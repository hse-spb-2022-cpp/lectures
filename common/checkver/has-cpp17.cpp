#include <iostream>
#include <vector>

int main() {
    std::cout << "Hello World!\n";
    std::vector vec{1, 2, 3, 4};  // Requires C++17.
    std::cout << "1 == " << vec[0] << "\n";
}
