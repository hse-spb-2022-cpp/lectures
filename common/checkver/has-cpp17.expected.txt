Hello World!
1 == 1
